### PR TITLE
Added more Lang Options

### PR DESCRIPTION
**Pull request by SoRadGaming**
resolves issue #18

### CODE REVIEW

File: src/me/ryandw11/ultrachat/commands/ChannelCmd.java [hunk 1]
```diff
@@ -66,19 +66,20 @@ else if(args.length == 0){
...
 			}
 			else if(p.hasPermission(plugin.channel.getString(args[0] + ".permission")) || plugin.channel.getString(args[0] + ".permission").equalsIgnoreCase("none")){
 				plugin.data.set(p.getUniqueId() + ".channel", args[0]);
 				plugin.saveFile();
-				p.sendMessage(ChatColor.BLUE + "You are now in the channel " + args[0] + "!");
+				p.sendMessage(Lang.CHANNEL_CHANGE.toString().replace("%s", args[0]));
 			}
 			else{
-				p.sendMessage(ChatColor.RED + "You do not have permission to join that channel.");
+				p.sendMessage(Lang.NO_PERM_JOIN.toString());
```
Comment (ryandw11) [inline src/me/ryandw11/ultrachat/commands/ChannelCmd.java:81]: This is not needed due to the message below this.

**Review by ryandw11 — CHANGES_REQUESTED** [review]
This looks good. I recommend removing the highlighted no_perm message though.

**Comment by SoRadGaming** [comment]
removed that from all three files, must have misliked and added it, not intended

**Review by ryandw11 — APPROVED** [review]
Looks good. Thanks for contributing!

**Pull request merged**